app/routes/project/index.js

```diff
@@ -10,15 +10,13 @@ export default Route.extend({
   userSubscriptions: service(),
 
   model() {
-    let project = this.modelFor('project');
-
-    return RSVP.hash({
-      project,
-      subscription: this.get('userSubscriptions').fetchForProject(project)
+    return this.modelFor('project').reload().then((project) => {
```

Comment:
@sbatson5 do you mean calling it from within here? Sorry I'm just not understanding where/how you would implement this specifically.